view/frontend/templates/breadcheckout/cartview.phtml

```diff
@@ -9,10 +9,11 @@
     <div class="button-prevent" id="button-prevent">
         <span class="buy_error_tip override_tip" data-content="<?= /* @noEscape */ $block->productTypeErrorMessage() ?>">&nbsp;</span>
     </div>
+    <div class="splitpay-clickable-button"></div>
 </div>
 
 <script type="text/javascript">
-    require(['jquery', 'jquery/validate', 'mage/validation'], function($, quote) {
+    require(['jquery', 'Bread_BreadCheckout/js/product/splitpay', 'jquery/validate', 'mage/validation'], function($, splitpay) {
```

Comment:
This can be placed in require-config.js and then use shorthand name, but not necessary, just an option.